packages/protocol/src/action/request/Invoke.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * @license
+ * Copyright 2022-2025 Matter.js Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { FALLBACK_INTERACTIONMODEL_REVISION } from "#session/Session.js";
+import { ClusterType, CommandData, InvokeRequest, TlvSchema, TypeFromSchema } from "#types";
+import { MalformedRequestError } from "./MalformedRequestError.js";
+import { Specifier } from "./Specifier.js";
+
+export interface Invoke extends InvokeRequest {
+    timeout?: number;
+}
+
+/**
+ * Request invocation of one or more commands.
+ */
+export function Invoke(definition: Invoke.Definition): Invoke {
+    let { interactionModelRevision, suppressResponse, timed: timedRequest } = definition;
+    const { commands } = definition;
+
+    if (!commands?.length) {
+        throw new MalformedRequestError(`Invocation requires at least one command`);
+    }
+
+    interactionModelRevision ??= FALLBACK_INTERACTIONMODEL_REVISION;
```

Comment:
you could add setting the fallback values in the deconstruction above then they could stay const

Reply:
See discord. Your adjustment is not what I meaned and will not work. Please adjust again.